src/main/java/frc/robot/subsystems/scoring/ScoringSubsystem.java

```diff
@@ -100,16 +128,41 @@ private void endgame() { // TODO: Later
         state = ScoringState.IDLE;
     }
 
-    private double findShootAngleRads() { // TODO: Interpolate
-        double distancetoGoal = Math.sqrt(Math.pow(Math.sqrt(Math.pow(1, 2) + Math.pow(1, 2)), 2) + Math.pow(1, 2));
-        return Math.atan2(distancetoGoal, 1);
+    private double findDistanceToGoal() {
+        Translation2d speakerPose =
+                false // TODO: CHANGE THIS URGENT
+                        // DriverStation.getAlliance().get() == DriverStation.Alliance.Red
+                        ? new Translation2d(
+                                Units.inchesToMeters(652.73), Units.inchesToMeters(218.42))
+                        : new Translation2d(
+                                Units.inchesToMeters(-1.5),
+                                Units.inchesToMeters(218.42)); // TODO: Might have to change these
```

Comment:
These could also be put into a `FieldConstants` in `Constants.java` in case other subsystems want to use them (drivetrain likely will)